src/GraphQLParser/Visitors/SDLPrinter.cs

```diff
@@ -1164,7 +1170,10 @@ private static async ValueTask WriteEncodedStringAsync(TContext context, ROM val
                 else if (code == '\t')
                     await context.WriteAsync("\\t").ConfigureAwait(false);
                 else
-                    await context.WriteAsync("\\u" + ((int)code).ToString("X4")).ConfigureAwait(false);
+                {
+                    await context.WriteAsync("\\u00").ConfigureAwait(false);
```

Comment:
We are under `code < ' '` condition and high byte is always `0`.

Reply:
Why not add "\\u00" to each string in the array so we eliminate a call to WriteAsync?

Reply:
done